pyproject.toml

```diff
@@ -152,19 +145,9 @@ gcp = [
     "google-cloud-tpu>=1.18.3",
     "dstack[server]",
 ]
-datacrunch = [
-    "datacrunch",
-    "dstack[server]",
-]
-kubernetes = [
-    "kubernetes",
-    "dstack[server]",
-]
-lambda = [
-    "boto3>=1.38.13",
-    "botocore",
-    "dstack[server]",
-]
+datacrunch = ["datacrunch", "dstack[server]"]
+kubernetes = ["kubernetes", "dstack[server]"]
+lambda = ["boto3>=1.38.13", "botocore", "dstack[server]"]
```

Comment:
Why formatting pyproject.toml differently? I like dependency per line as it was.